.github/actions/validate-with-schema/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Validate With Schema
+description: Action that validates json/yaml data against a given schema
+inputs:
+  schema-version:
+    type: string
+    required: true
+    description: Version of the schema required, in SchemaVer
+  schema-location:
+    type: string
+    required: true
+    description: Directory within access-nri/schema that contains the schema
+  data-location:
+    type: string
+    required: true
+    description: Path to the data in the callers repository
+runs:
+  using: composite
+  steps:
+      # Checkout the callers `spack.yaml`
+    - uses: actions/checkout@v4
+
+    # Checkout the `schema-repo`
+    - uses: actions/checkout@v4
+      with:
+        repository: access-nri/schema
+        path: schema
+
+    - name: Check data extension
+      shell: bash
+      id: data
+      run: |
+        if [[ "${{ inputs.data-location }}" ~= \.ya?ml$ ]]; then
+          echo "yaml=true" >> $GITHUB_OUTPUT
+        else
+          echo "yaml=false" >> $GITHUB_OUTPUT
+        fi
+
+    - uses: GrantBirki/json-yaml-validate@d7814b94473939c1daaca2c96131b891d4703a3c # v2.7.1
+      with:
+        mode: fail
+        files: |
+          ${{ inputs.spack-yaml-location }}
```

Comment:
This is `data-location` in the inputs.

Can it handle more than one file?

Reply:
Updated variable name. And yes, it can take multiple files...I'll make that explicit

Reply:
It can, specified that in the inputs now